robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -89,43 +90,64 @@ public static final class ELEVATOR {
 
     public static final double GEAR_RATIO = 50 / 14;
     public static final Distance MOTOR_PULLEY_PITCH_DIAMETER = Units.Inches.of(
-      2.256
-    );
+        2.256);
 
     public static final double AXIS_MAX_SPEED = 0.1;
 
     public static final Distance[] ELEVATOR_HEIGHT_SETPOINTS = {};
   }
 
   public static final class ALGAE_RUNNER {
+    public static final double AXIS_MAX_SPEED = 0.8;
+
+    public static final IdleMode IDLE_MODE = IdleMode.kBrake;
+
+    public static final boolean MOTOR_INVERTED = false;
+
+    public static final Current MOTOR_STALL_LIMIT = Units.Amps.of(50);
+    public static final Current MOTOR_FREE_LIMIT = Units.Amps.of(50);
+
+    public static final double RAMP_RATE = .25;
+
     public static final SparkBaseConfig MOTOR_CONFIG = new SparkMaxConfig()
-      .idleMode(IdleMode.kBrake)
-      .inverted(false);
-    
-    public static final double LEFT_MOTOR_P = 0;
-    public static final double LEFT_MOTOR_I = 0;
-    public static final double LEFT_MOTOR_D = 0;
-    public static final double LEFT_MOTOR_F = 0;
+        .idleMode(IDLE_MODE)
+        .inverted(MOTOR_INVERTED)
+        .smartCurrentLimit((int)MOTOR_STALL_LIMIT.in(Units.Amps), (int)MOTOR_FREE_LIMIT.in(Units.Amps));
+  }
 
-    public static final ClosedLoopConfig CLOSED_LOOP_CONFIG_LEFT =
-      MOTOR_CONFIG.closedLoop
-        .pidf(LEFT_MOTOR_P, LEFT_MOTOR_I, LEFT_MOTOR_D, LEFT_MOTOR_F)
-        .outputRange(-1, 1);
+  public static final class ALGAE_PIVOT {
```

Comment:
We will need a closedloop controller for the pivot like you originally had for the roller